common/test/acceptance/pages/studio/settings.py

```diff
@@ -61,8 +63,13 @@ def save_changes(self, wait_for_confirmation=True):
                 'Waiting for save confirmation...'
             ).fulfill()
 
-    def refresh_page(self):
+    def refresh_page(self, wait_for_confirmation=True):
```

Comment:
The name wait_for_confirmation is confusing to me, it implies that you're waiting for a confirmation dialog to appear. That's not the case, right, you're waiting for the "Saving changes..." popup to disappear? I'm not sure.